templates/apache-fragment.epp

```diff
@@ -0,0 +1,60 @@
+<%- |
```

Comment:
As noted yesterday, the approach taken in https://github.com/puppetlabs/puppetlabs-apache/pull/1659 feels better to me. I've opened https://github.com/puppetlabs/puppetlabs-apache/pull/2167 which is where I think we should go. Having that could also clean up the puppet-pulpcore module.

Reply:
I've opened https://github.com/ekohl/puppetlabs-apache/pull/1 to get us closer to having a working version